recipes/taocpp-json/1.0.0-beta.10/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.0-beta.10":
+    sha256: c6e8eb2b3a1f8dd284e02303527d26c5856cac24e67f1f5a8fc622b892e2cd26
+    url: https://github.com/taocpp/json/archive/1.0.0-beta.10.tar.gz
```

Comment:
Will there be soon-ish a stable version? We don't normally add pre-release versions

Reply:
Well, I am not the maintainer of the package. we consider the beta.x normal releases, we use the beta-10 version for a couple of years. We are stuck with C++11 compiler, otherwise we would move to later version.

Reply:
Besides Conan, I'm also maintain Taocpp. I'll talk to Daniel Frey to check if he will release a 1.0.0 soon or not.

Reply:
No stable release in mind for now, only betas. So we still need to support beta packages ...

Reply:
@uilianries  thanks for helping out on getting this change into conan.